### PR TITLE
Add test against PHP 8.2 & update dev dependencies

### PR DESCRIPTION
**Pull request by pascalheidmann**

### CODE REVIEW

**Comment by pascalheidmann** [comment]
@func0der pls take a look :)

File: composer.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
         "psr-7"
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
```
Comment (func0der) [inline composer.json:32]: So we can only support PHP8.2 if we kick 7.4?

Comment (func0der) [reply composer.json:32]: I was just wondering, but since PHP7.4 is EOL I will not force this compatibility.

**Comment by func0der** [comment]
Thank you for your contribution :)

**Review by func0der — APPROVED** [review]

**Pull request merged**